mage_ai/settings/server.py

```diff
@@ -73,7 +73,11 @@ def get_bool_value(value: str) -> bool:
 HOSTNAME = os.getenv('HOSTNAME')
 REDIS_URL = os.getenv('REDIS_URL')
 SERVER_VERBOSITY = os.getenv('SERVER_VERBOSITY', 'info') or 'info'
-SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT')
+SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT', 'plaintext')
+SERVER_LOGGING_TEMPLATE = os.getenv(
+    'SERVER_LOGGING_TEMPLATE',
+    '[%(levelname)s]:[%(name)s]:%(message)s',
```

Comment:
can we set the default to match what the default was before this change?

before: `INFO:mage_ai.server.scheduler_manager:Scheduler status: running.`
after: `[INFO]:[mage_ai.server.scheduler_manager]:Scheduler status: running.`

Reply:
Sure! I copied it from docs https://docs.mage.ai/development/observability/logging#server-logging-format - maybe fix it here as well?

Reply:
Done

Reply:
oops, good catch. i can update that